UM/Mesh/MeshData.py

```diff
@@ -244,6 +244,30 @@ def reserveVertexCount(self, num_vertices):
         self._vertex_count = 0
         self._face_count = 0
     
+    ##  Set the amount of faces and vertices before loading data to the mesh.
+    #
+    #   This way we can create the array before we fill it. This method will reserve
+    #   `num_vertices` amount of space for vertices, `num_vertices` amount of space
+    #   for colors and `num_faces` amount of space for indices.
+    #
+    #   \param num_faces Number of faces for which memory must be reserved.
+    #   \param num_vertices Number of vertices for which memory must be reserved.
+    def reserveFaceAndVerticeCount(self, num_faces, num_vertices):
+        if type(num_faces) == float:
+            Logger.log("w", "Had to convert 'num_faces' with int(): %s -> %s ", num_faces, int(num_faces))
+            num_faces = int(num_faces)
+        if type(num_vertices) == float:
```

Comment:
For type checking, it is better to use isinstance(num_vertices, float) since that supports subclasses. However, in general it is better not to typecheck in python. If you want to ensure they are ints, just call int(num_vertices) when using it. That way anything that can be converted to an int is supported instead of special casing only for float.